src/components/MoneyRequestConfirmationList.tsx

```diff
@@ -506,10 +506,9 @@ function MoneyRequestConfirmationList({
                     hideCurrencySymbol
                     formatAmountOnBlur
                     prefixContainerStyle={[styles.pv0]}
-                    inputStyle={
-                        [styles.optionRowAmountInput, StyleUtils.getPaddingLeft(StyleUtils.getCharacterPadding(currencySymbol ?? '') + styles.pl1.paddingLeft), amountWidth] as TextStyle[]
-                    }
+                    inputStyle={[styles.optionRowAmountInput, amountWidth] as TextStyle[]}
                     containerStyle={[styles.textInputContainer]}
+                    touchableInputWrapperStyle={[StyleUtils.getPaddingLeft(StyleUtils.getCharacterPadding(currencySymbol ?? '') + styles.pl1.paddingLeft)]}
```

Comment:
Can you revert this and just add `touchableInputWrapperStyle={[styles.ml3]}`

Reply:
Yes, but I think left padding is not necessary inside the `inputStyle` (nothing changes with or without)

Reply:
Right that style is already added in TextInput
https://github.com/Expensify/App/blob/a6cb60c63201e823406cee5298b682be796a804b/src/components/TextInput/BaseTextInput/index.tsx#L365

Reply:
So should I revert the change or not? `styles.ml3` is not the same as the formula (for `PLN` left padding is `20`)

Reply:
In staging there is a fixed margin of 12px, let's go with that